README.md

```diff
@@ -91,7 +91,30 @@ We follow the official OASIS specifications in order to provide as much acceptan
 
 ### Developer Guide, Architecture and Technical Design
 
-``` To Be Published ```
+The converter uses lxml.objectify to parse the whole input document.
+
+Parsing and conversion of the following CVRF XML elements are handled by separate section handlers. These section handlers process the elements recursively (converting also all their sub-elements). These elements are the direct children of the root XML element (`<cvrfdoc>`).
+ - Acknowledgments -> `Acknowledgments` handler
+ - DocumentNotes -> `Notes` handler
+ - DocumentPublisher -> `DocumentPublisher` handler
+ - DocumentReferences -> `References` handler
+ - DocumentTracking -> `DocumentTracking` handler
+ - ProductTree -> `ProductTree` handler
+ - Vulnerability -> `Vulnerability` handler
+ - DocumentType, DocumentTitle, DocumentDistribution, AggregateSeverity -> `DocumentLeafElements` handler
```

Comment:
Please sort by [CSAF CVRF 1.2 order](http://docs.oasis-open.org/csaf/csaf-cvrf/v1.2/cs01/csaf-cvrf-v1.2-cs01.html#_Toc493508866) and link the corresponding files for the handlers (as shown for `DocumentLeafElements` - please make sure the relative links actually work):
```suggestion
 - DocumentTitle, DocumentType, DocumentDistribution, AggregateSeverity -> [`DocumentLeafElements`](blob/main/cvrf2csaf/section_handlers/document_leaf_elements.py) handler
 - DocumentPublisher -> `DocumentPublisher` handler
 - DocumentTracking -> `DocumentTracking` handler
 - DocumentNotes -> `Notes` handler
 - DocumentReferences -> `References` handler
 - Acknowledgments -> `Acknowledgments` handler
 - ProductTree -> `ProductTree` handler
 - Vulnerability -> `Vulnerability` handler